day_10/task_1/tests/pipe_maze_tests.cpp

```diff
@@ -1,9 +1,23 @@
 #include "day_10/task_1/pipe_maze.hpp"
 #include "gtest/gtest.h"
 
-TEST(PipeMazeToolsShould, FindTheStartingPoint)
+
+struct PipeMazeTests : public ::testing::Test 
 {
     const PipeMaze pipe_maze = {"-L|F7", "7S-7|", "L|7||", "-L-J|", "L|-JF"};
+};
+
+TEST_F(PipeMazeTests, FindTheStartingPoint)
+{
+    ASSERT_EQ(find_starting_point(pipe_maze), MazePoint(1, 1));
+}
+
+TEST_F(PipeMazeTests, GivenStartingPipe_WhenFindNextNeighbors_ExpectTwoMazePointsOfConnectedPipes)
+{
+    const MazePoint starting_pipe(find_starting_point(pipe_maze));
+
+    const auto foundNeighbors = find_next_neighbors(pipe_maze, starting_pipe);
```

Comment:
Could you allign to style one common style these two names, please?

Reply:
done